.github/workflows/e2e-tests-on-PR.yml

```diff
@@ -0,0 +1,33 @@
+name: PR->master
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  cypress:
+    name: "Cypress"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: launch the world
+      run: make install && docker-compose up -d
```

Comment:
Probably not a thing for today, but may we worth investigating if the build is faster in Docker Compose or native on GH Actions (Docker Compose is definitely simpler)